portal/portal-sprint-98.md

```diff
@@ -0,0 +1,57 @@
+---
+slug: /release-notes/portal/sprint-98
+title: Sprint 98 (April 19, 2021)
+---
+
+# fiskaltrust.Portal - Sprint 98
+_April 19, 2021_
+
+**Introduce tool for generating AO148 applications**
+
+In this sprint, we have been focusing on creating a tool that should give users the ability to easily create AO 148 applications.
+
+## Features
+
+### Middleware Configuration
+- [Reworked Queue DE page (DE)](#reworked-queue-de-page-de)
+- [Reworked Queue DE SCU connection page (DE)](#reworked-queue-de-scu-connection-page-de)
+
+### User Management
+- [Reworked Outlets page](#reworked-outlets-page)
+- [Added email verification field to registration dialog](#added-email-verification-field-to-registration-dialog)
+
+### Tools
+- [Introduce tool for generating AO148 applications (DE)](#introduce-tool-for-generating-ao148-applications-de)
+ 
+## Features
+ 
+## Middleware Configuration
+ 
+### Reworked Queue DE page (DE)
+The Queue DE page has been reworked. Even though it appears to be similar to the old page, offering the same functionalities, response times of the page are faster and the overall data overview is improved.
+ 
+### Reworked Queue DE SCU connection page (DE)
+The Queue DE SCU connection page has been reworked. Even though it appears to be similar to the old page, offering the same functionalities, response times of the page are faster and the overall data overview is improved.
+ 
+## User Management
+ 
+### Reworked Outlets page
+The Outlets page has been reworked. Even though it appears to be similar to the old page, offering the same functionalities, response times of the page are faster and the overall data overview is improved.
```

Comment:
These three points say exactly the same. We should sum them up. e.g.:
## Improving Page Performance
We invested some time in improving how our tables are displayed. Visually nothing major should have changed but we managed to reduce updates on the tables and reduced necessary memory.
A wider range of monitor resolutions should now also be supported.
Affected by this change are the tables under the following views: Queue, ....